### ISSUE TITLE
Tab character (\t) in Dataframe causes JSON.parse error

### ISSUE THREAD

**Issue by jorisroovers** [Kanaries/pygwalker#17]
Thanks for creating pygwalker, just starting playing with it, it looks great!

When trying the following snippet:
```python
import pandas as pd
import pygwalker as pyg

df = pd.DataFrame([{"a":"\tb"}])
gwalker = pyg.walk(df)
```

I get this error:
```sh
Javascript error adding output!
SyntaxError: JSON.parse: bad control character in string literal at line 1 column 24 of the JSON data
See your browser Javascript console for more details.
```

**Comment by Asm-Def** [comment]
It seems to have been fixed in #31 and pre-released in `0.1.4-alpha2`.

Could you please have a try with `pip install "pygwalker>=0.1.4-alpha2"` and leave some feedback?

BTW: Welcome to join our discord <a href="https://discord.gg/Z4ngFWXz2U" rel="nofollow"><img alt="discord invitation link" src="https://dcbadge.vercel.app/api/server/Z4ngFWXz2U?style=flat" style="max-width: 100%;" align="middle"></a>  to keep track of the latest progress.

**Comment by jorisroovers** [comment]
> Could you please have a try with pip install "pygwalker>=0.1.4-alpha2" and leave some feedback?

Yup, that fixed it!
